scripts/run.sh

```diff
@@ -1,16 +1,160 @@
 #!/bin/bash
 
+[ "$DB_NAME" ] || DB_NAME='wordpress'
+[ "$DB_PASS" ] || DB_PASS='root'
+[ "$SEARCH_REPLACE" ] && \
+	BEFORE_URL=$(echo "$SEARCH_REPLACE" | cut -d ',' -f 1) && \
+	AFTER_URL=$(echo "$SEARCH_REPLACE" | cut -d ',' -f 2) \
+|| SEARCH_REPLACE=false
+
+# Error handling
+ERROR () { echo -e "\n=> $(tput setaf 1)$(tput bold)ERROR$(tput sgr 0) (Line $1): $2."; exit 1; }
+
+# Configure wp-cli
+if [ ! -f /app/wp-cli.yml ]; then
+	cat > /app/wp-cli.yml <<-EOF
+	quiet: true
+	apache_modules:
+	  - mod_rewrite
+
+	core config:
+	  dbuser: root
+	  dbpass: $DB_PASS
+	  dbname: $DB_NAME
+	  dbhost: db:3306
+
+	core install:
+	  url: $([ "$AFTER_URL" ] && echo "$AFTER_URL" || echo localhost:8080)
+	  title: $DB_NAME
+	  admin_user: root
+	  admin_password: $DB_PASS
+	  admin_email: admin@${DB_NAME}.com
+	  skip-email: true
+	EOF
+fi
+
+# Pause until MySQL is available for connections
+printf "=> Waiting for MySQL to initialize... \n"
+while ! mysqladmin ping --host=db --password=$DB_PASS --silent; do
+	sleep 1
+done
+
+# Configure WordPress
+printf "\t%s\n" \
+	"=======================================" \
+	"	Begin WordPress Configuration" \
+	"======================================="
+
 if [ ! -f /app/wp-config.php ]; then
-  /scripts/setup-wordpress.sh
+
+	# wp.config.php
+	printf "=> Generating wp.config.php file... "
+	sudo -u www-data wp core config >/dev/null 2>&1 || ERROR $LINENO "Could not generate wp-config.php file"
+	printf "Done!\n"
+
+else
+	printf "=> wp-config.php exists. SKIPPING...\n"
+fi
+
+# Install Database
+if [ ! "$(wp core is-installed --allow-root >/dev/null 2>&1 && echo $?)" ]; then
+
+	printf "=> Creating database '%s'... " "$DB_NAME"
+	sudo -u www-data wp db create >/dev/null 2>&1 || ERROR $LINENO "Database creation failed"
+	printf "Done!\n"
+
+	# If an SQL file exists in /data => load it
+	if [ "$(stat -t /data/*.sql >/dev/null 2>&1 && echo $?)" ]; then
+
+		DATA_PATH=$(find /data/*.sql | head -n 1)
+		printf "=> Loading data backup from %s... " "$DATA_PATH"
+		sudo -u www-data wp db import "$DATA_PATH" >/dev/null 2>&1 || ERROR $LINENO "Could not import database"
+		printf "Done!\n"
+
+		# If SEARCH_REPLACE is set => Replace URLs
+		if [ "$SEARCH_REPLACE" != false ]; then
+
+			printf "=> Replacing URLs... "
+			REPLACEMENTS=$(sudo -u www-data wp search-replace "$BEFORE_URL" "$AFTER_URL" \
+				--no-quiet --skip-columns=guid | grep replacement) || \
+				ERROR $((LINENO-2)) "Could not execute SEARCH_REPLACE on database"
+			echo -ne "$REPLACEMENTS\n"
+
+		fi
+
+	else
+		printf "%s\n" \
+			"=> No database backup found. Initializing new database..." " " \
+			"	  DEFAULT DATABASE CREDENTIALS" \
```

Comment:
I would either move L90-94 to L66 (aka showing all the time at setup) or remove it altogether (if we use this in production, logging the password is not ideal). I would probably remove it since it doesn't serve any purpose, we set the wp-config.php for you and you can see these in it.